services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java

```diff
@@ -555,6 +562,14 @@ public Response checkFileInfo(@Context UriInfo uriInfo, @Context HttpServletRequ
                      .entity("{\"error\": \"" + e.getMessage() + "\"}")
                      .type(MediaType.APPLICATION_JSON)
                      .build();
+    } catch (RepositoryException e) {
+      if (LOG.isDebugEnabled()) {
+        LOG.debug("Cannot check file info", e);
+      }
+      return Response.status(Status.INTERNAL_SERVER_ERROR)
+                     .entity("{\"error\": \"Cannot check file info\"}")
```

Comment:
If it's server error, then not "Cannot" but "Error" or "Failed to". A "Cannot" is good for client errors.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/12/commits/1dbc50655f9e4081bde0523b78ad7e9e73bd36bd